data-otservbr-global/lib/core/storages.lua

```diff
@@ -893,7 +892,6 @@ Storage = {
 		DeeplingAnchor = 51023,
 		FirstOrientalAddon = 51024,
 		SecondOrientalAddon = 51025,
-		RoyalCostumeOutfit = 51026,
```

Comment:
```suggestion
		-- RoyalCostumeOutfit = 51026, (moved to kv)
```

Reply:
there is no need, as it is free storage, it was set to -1 in the migration